config/initializers/carrierwave.rb

```diff
@@ -1,3 +1,11 @@
+require "mini_magick"
+
+# Carrierwave uses MiniMagick for image processing. To prevent server tmeouts
+# we are setting the MiniMagick timeout lower.
+MiniMagick.configure do |config|
+  config.timeout = 10
```

Comment:
Totally open to other timeout values if anyone has a suggestion!

Reply:
ahahha if it can't open an image in 10 seconds we should definitely bail :D